GitHub rate limiter causing defect raising to fail

See, for example, https://github.com/quarkusio/extensions/actions/runs/8087944319/job/22100951380

```
org.kohsuke.github.GHException: Failed to retrieve https://api.github.com/search/issues?q=is%3Aissue+is%3Aopen+%22https%3A%2F%2Fcamel.apache.org%2F_%2Fimg%2Flogo-d-f21b25ba38.svg%22+in%3Atitle+repo%3Aquarkusio%2Fextensions
	at org.kohsuke.github.GitHubPageIterator.fetch(GitHubPageIterator.java:153)
	at org.kohsuke.github.GitHubPageIterator.hasNext(GitHubPageIterator.java:89)
	at org.kohsuke.github.PagedSearchIterable$1.hasNext(PagedSearchIterable.java:86)
	at org.kohsuke.github.PagedIterator.fetch(PagedIterator.java:106)
	at org.kohsuke.github.PagedIterator.hasNext(PagedIterator.java:74)
	at org.kohsuke.github.PagedSearchIterable.populate(PagedSearchIterable.java:65)
	at org.kohsuke.github.PagedSearchIterable.getTotalCount(PagedSearchIterable.java:49)
	at Report.processBadImage(bad-image-issue.java:141)
	at Report.lambda$run$0(bad-image-issue.java:66)
	at java.base/java.util.Arrays$ArrayList.forEach(Arrays.java:4204)
	at Report.run(bad-image-issue.java:66)
	at picocli.CommandLine.executeUserObject(CommandLine.java:1769)
	at picocli.CommandLine.access$900(CommandLine.java:145)
	at picocli.CommandLine$RunLast.executeUserObjectOfLastSubcommandWithSameParent(CommandLine.java:2150)
	at picocli.CommandLine$RunLast.handle(CommandLine.java:2144)
	at picocli.CommandLine$RunLast.handle(CommandLine.java:2108)
	at picocli.CommandLine$AbstractParseResultHandler.execute(CommandLine.java:1975)
	at picocli.CommandLine.execute(CommandLine.java:1904)
	at Report.main(bad-image-issue.java:195)
Caused by: org.kohsuke.github.HttpException: {
  "documentation_url": "https://docs.github.com/free-pro-team@latest/rest/overview/rate-limits-for-the-rest-api#about-secondary-rate-limits",
  "message": "You have exceeded a secondary rate limit. Please wait a few minutes before you try again. If you reach out to GitHub Support for help, please include the request ID B1E9:7941:2692D9A:4D222D1:65DFB44A."
}
	at org.kohsuke.github.GitHubConnectorResponseErrorHandler$1.onError(GitHubConnectorResponseErrorHandler.java:56)
```

--- a/site-validation/bad-image-issue.java
+++ b/site-validation/bad-image-issue.java
@@ -2,7 +2,7 @@
 
 //JAVA 17+
 
-//DEPS org.kohsuke:github-api:1.321
+//DEPS org.kohsuke:github-api:1.322
 //DEPS info.picocli:picocli:4.2.0
 
 import com.fasterxml.jackson.core.JsonProcessingException;
@@ -144,7 +144,7 @@ class Report implements Runnable {
                         %s image: %s
 
                         The problem image was found on these artifacts: %s
-                        
+                                                
                         Affected pages are 
                         - %s
 
@@ -183,12 +183,12 @@ class Report implements Runnable {
         if (Files.exists(filePath)) {
             try {
                 return Arrays.asList(new ObjectMapper().readValue(filePath.toFile(), BadImage[].class));
-                            } catch (JsonProcessingException e) {
-                                throw new RuntimeException(e);
-                            }
-            } else {
-                return Collections.emptyList();
+            } catch (JsonProcessingException e) {
+                throw new RuntimeException(e);
             }
+        } else {
+            return Collections.emptyList();
+        }
     }
 
     public static void main(String... args) {

--- a/site-validation/dead-link-issue.java
+++ b/site-validation/dead-link-issue.java
@@ -18,7 +18,7 @@
 
 //JAVA 17+
 
-//DEPS org.kohsuke:github-api:1.321
+//DEPS org.kohsuke:github-api:1.322
 //DEPS info.picocli:picocli:4.2.0
 
 import com.fasterxml.jackson.core.JsonProcessingException;

